compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -337,12 +345,41 @@ TR::Register *OMR::ARM64::CodeGenerator::gprClobberEvaluate(TR::Node *node)
       }
    }
 
+static uint32_t registerBitMask(int32_t reg)
+   {
+   return 1 << (reg-1);
+   }
 
 void OMR::ARM64::CodeGenerator::buildRegisterMapForInstruction(TR_GCStackMap *map)
    {
-   TR_UNIMPLEMENTED();
-   }
+   TR_InternalPointerMap * internalPtrMap = NULL;
+   TR::GCStackAtlas *atlas = self()->getStackAtlas();
```

Comment:
Just FYI, see #3932.

Reply:
Ok given #3932 is going to implement this; shall I remove it from this platform specific PR?

Reply:
We need some implementation in AArch64.  I'll accept the first one that is ready, so leave it in for now.  If your PR lands first then #3932 will have to remove it.  If #3932 lands first then I'll ask you to remove your implementation before merging.

Reply:
Ok. Sounds good to me. I'm just verifying local compilation and Tril tests now. Then I'll push an updated version of this PR.